<launch>
  <env name="ROSCONSOLE_FORMAT" value="[${severity}][${thread}][${node}/${function}:${line}]: ${message}"/>

<arg name="posearray_topic" default="/waypoints"/>

  <node pkg="follow_waypoints" type="follow_waypoints" name="follow_waypoints" output="screen" clear_params="true">
    <param name="goal_frame_id" value="map"/>
    <param name="posearray_topic" value="$(arg posearray_topic)"/>
    <param name="waypoint_distance_tolerance" value= "0.5"/>
  </node>

  <node pkg="follow_waypoints" type="path_planner" name="pathplanner" output="screen"/>

</launch>
